pkg/tui/log_core.go

```diff
@@ -47,7 +47,7 @@ func (c *LogCore) With(f []zapcore.Field) zapcore.Core {
 			if c.verbosity.CombineLogs() {
 				field.AddTo(nc.enc)
 			}
-			// else (if the field is the construct and we're not combining logs) don't add it to the encoder
+			// else (if the field is the construct and we're not combining errors) don't add it to the encoder
```

Comment:
Combing logs is the correct word here. This describes the difference, for eg, between `-v=2` and `-v=3`